.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish Package
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python package
+    runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish package to test PyPI
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Maybe changed this line to check for the presence of the word `test` in the tag name and line 37 to exclude it, so it would only trigger a test publish if we did `git tag test_1.23.45` for example

Reply:
Can you give it a try..?

Reply:
I would suggest only merging/testing the pypi test part for now by making a new release/tag and check if it's correctly published on test pypi. If it is, uncomment the release pypi in master and assume it will be fine when another release is made.